### PR TITLE
Add animated plots for ScatterGeodesicsPlotter.

### LINKED ISSUE #36: Simulation of orbits - Animations
Simulating orbits with coordinate time running and correspondingly plot radial and angular coordinates and showing plots in animation format

### PR DESCRIPTION
**Pull request by Varunvaruns9**
Fixes parts of #36 

This PR adds an animated plotting system for simulation of orbits. Currently, I've only implemented it for ScatterGeodesicsPlotter. I'll add the tests and a similar method for Static plotter and maybe Kerr and Kerr-Newman (when they are added), once this gets a green light.

Here is what I have been able to get-
Screen-Recording-2019-04-10-at-8 49 37-PM

@shreyasbapat @spino17 FYI.

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Yess! Well, this is how the plotting ecosystem should grow! :) 
Now I would like to see some controllable animation!

**Comment by shreyasbapat** [comment]
And BTW All lights green! I will review thoroughly in a moment

**Comment by Varunvaruns9** [comment]
Updated the tests and added a new feature to control the speed of plotting. @shreyasbapat

**Comment by ritzvik** [comment]
Really great work @Varunvaruns9 .

**Comment by Varunvaruns9** [comment]
Thanks @ritzvik 😀

**Comment by shreyasbapat** [comment]
My god! What happened to codecov?

**Comment by shreyasbapat** [comment]
I see : https://status.codecov.io/ 
We can't merge this before we get coverage report :/

**Comment by Varunvaruns9** [comment]
Please merge #180 before this. I would have to resolve some conflicts before this is mergeable.

**Comment by Varunvaruns9** [comment]
Resolved conflicts.

File: src/einsteinpy/plotting/geodesics_scatter.py [hunk 1]
```diff
@@ -40,6 +41,53 @@ def plot(self, pos_vec, vel_vec, end_lambda=10, step_size=1e-3):
         if not self._attractor_present:
             self._plot_attractor()
 
+    def plot_animated(
```
Comment (shreyasbapat) [inline src/einsteinpy/plotting/geodesics_scatter.py]: This method must only be named `animate` .

File: src/einsteinpy/plotting/geodesics_scatter.py [hunk 2]
```diff
@@ -40,6 +41,53 @@ def plot(self, pos_vec, vel_vec, end_lambda=10, step_size=1e-3):
...
+        self, pos_vec, vel_vec, end_lambda=10, step_size=1e-3, interval=50
+    ):
+        """
+
+        Parameters
+        ----------
+        Interval: Control the time between frames. Add time in milliseconds.
+
+        Other parameters are same as plot method.
```
Comment (shreyasbapat) [inline src/einsteinpy/plotting/geodesics_scatter.py]: Please mention everything. Moreover, I see `interval` in the function declaration. Also, The proper way of writing docstring would be: 

```
Parameters
----------------
interval : int, optional
    Control the time between frames. Add time in milliseconds.
```

**Comment by Varunvaruns9** [comment]
@shreyasbapat Done the changes. Thanks for the review :)

File: src/einsteinpy/plotting/geodesics_scatter.py [hunk 3]
```diff
@@ -40,6 +56,59 @@ def plot(self, pos_vec, vel_vec, end_lambda=10, step_size=1e-3):
...
+        pic = plt.scatter([], [], s=1)
+        plt.scatter(0, 0, color="black")
+
+        def _update(frame):
+            pic.set_offsets(np.vstack((pos_x[: frame + 1], pos_y[: frame + 1])).T)
+            pic.set_array(time[: frame + 1])
+            return (pic,)
+
+        ani = FuncAnimation(fig, _update, frames=frames, interval=interval)
+        plt.show()
```
Comment (shreyasbapat) [inline src/einsteinpy/plotting/geodesics_scatter.py]: Remove this line. We already have it in `show` method :)

Comment (Varunvaruns9) [reply src/einsteinpy/plotting/geodesics_scatter.py]: The animation does not work with the class' `show` method, it needs `plt.show()` in that function. I will have to find some other way to make it work.

Comment (Varunvaruns9) [reply src/einsteinpy/plotting/geodesics_scatter.py]: Fixed.

**Comment by Varunvaruns9** [comment]
@shreyasbapat Fixed.

**Comment by shreyasbapat** [comment]
One rebase and it's in
Screenshot from 2019-04-12 02-50-27

**Comment by shreyasbapat** [comment]
Thanks @Varunvaruns9 :)

**Pull request merged**